src/main/kotlin/io/github/zebrateam/ZebraApplication.kt

```diff
@@ -0,0 +1,12 @@
+package io.github.zebrateam
+
+import org.springframework.boot.SpringApplication
+import org.springframework.boot.autoconfigure.SpringBootApplication
+
+
+@SpringBootApplication
+class ZebraApplication
+
+fun main(args: Array<String>) {
+    SpringApplication.run(ZebraApplication::class.java, *args)
+}
```

Comment:
About newlines. I will set my IDE to automatically add it (if there is such option).